yt/frontends/ramses/data_structures.py

```diff
@@ -369,8 +444,11 @@ def _detect_output_fields(self):
         dsl = set([])
         if self.fluid_field_list is None or len(self.fluid_field_list) <= 0:
             self._setup_auto_fields()
+
         for domain in self.domains:
             dsl.update(set(domain.particle_field_offsets.keys()))
+            dsl.update(set(domain.sink_field_offsets.keys()))
```

Comment:
In principle it would be nice to avoid having a line here for all the different particle types yt supports, but given that there's only two we can come back to that if we ever want to support different kinds of particles with completely different I/O (hopefully nothing like that actually exists).

Reply:
In general agreed, we'll have to think about it if a new particle output appears!